ptw/ptw_payload.py

```diff
@@ -0,0 +1,263 @@
+if last_attended_sec:
+    delta_sec = int(current_time_sec - float(last_attended_sec))
+    days_elapsed = delta_sec // (24 * 60 * 60)
+    hours = delta_sec % (24 * 60 * 60) // 3600
+
+    out.append(f'-f "Last Attended|{hours} hour(s) \
+                and {days_elapsed} day(s) ago."')
+
+    pay_cuts = days_elapsed // DAYS_PER_PAY_CUT
+
+    if pay_cuts > 0 and job_details[FIELD_WAGE]:
+        previous_wage = job_details[FIELD_WAGE]
+        job_details[FIELD_WAGE] -= pay_cuts
+
+        out.append(f'-f "Nonattendance|Because of nonattendance, \
+                            your wages have dropped from \
+                            {previous_wage} to {job_details[FIELD_WAGE]}."')
+else:
+    out.append(f'-f "Last Attended|This is your first day on the job!"')
+
+job_details[FIELD_LAST_ATTENDED] = current_time_sec
+
+
+# Check
+new_check = parsed_args.last(ARG_CHECK)
+
+if new_check:
+    matches = [x for x in SKILL_NAMES if new_check.lower() in x.lower()]
+    check = matches[0] if matches else None
+
+    if check:
+        job_details[FIELD_CHECK] = check
+        job_modifications.append(f'Skill check set to {new_check}')
+    else:
+        errors.append(f'-f "Err: Invalid Check|`{new_check}` is not a \
+                        valid value for `-{ARG_CHECK}`.\n\
+                        Try again with the name of a skill or ability.\n\n\
+                        Examples:\n\
+                        `!{ALIAS_NAME} modest -{ARG_CHECK} performance`\n\
+                        (sets your check to performance; will automatically \
+                        include proficiency/expertise)\n\n\
+                        `!{ALIAS_NAME} modest -{ARG_CHECK} wisdom -{ARG_BONUS} 2`\n\
+                        (Use this version if you are working \
+                        with a tool check. Sets your check to Wisdom, \
+                        with a flat additional \
+                        bonus of 2; you would use this \
+                        to represent, for example, a +2 proficiency bonus.)"')
+
+new_bonus = parsed_args.last(ARG_BONUS)
+
+if new_bonus:
+    job_details[FIELD_BONUS] = int(new_bonus)
+    job_modifications.append(f'Check bonus set to {new_bonus}')
+
+check_name = job_details.get(FIELD_CHECK)
+if check_name and not errors and job_details[FIELD_WAGE] and job_details[FIELD_WAGE] > 0:
+
+    # TODO: turn adv into a setting, so it doesn't need to be passed every time.
+    # adv_dice can technically be 2 if ea is passed, but...
+    # Uhh. People shouldn't be doing that?
+    adv_dice = parsed_args.adv()
+    desc_builder.append(f'**Base Wage:** {job_details[FIELD_WAGE]}')
+    desc_builder.append(f'**Checking:** {check_name}{" + " + str(job_details[FIELD_BONUS]) if job_details[FIELD_BONUS] else ""}{" with advantage" if adv_dice > 0 else ""}')
+
+    check_bonus = get_raw().skills.get(check_name)
+    if job_details[FIELD_BONUS]:
+        manual_bonus = f' + {job_details[FIELD_BONUS]}'
+    else:
+        manual_bonus = ""
```

Comment:
This can be rewritten to a ternary
```suggestion
        manual_bonus = f' + {job_details[FIELD_BONUS]}' if job_details[FIELD_BONUS] else ""
```

Reply:
- [ ] this